tests/ColorThiefTest.php

```diff
@@ -18,6 +18,11 @@ public function provideImageDominantColor()
                 null,
                 array(107, 172, 222)
             ),
+            array(
+                "/images/bookcover.jpg",
```

Comment:
I am not a (copyright) lawyer, but some [cursory](https://www.copyrightlaws.com/legally-using-images/) [reading](https://fairuse.stanford.edu/overview/fair-use/four-factors/) leads me to believe that the use of the image in this test case falls under "fair use", primarily because the inclusion of the image in no conceivable way *deprives the copyright owner of income or undermines a new or potential market for the copyrighted work*. 

Additionally, the image is already available online at numerous [e-commerce](https://www.imusic.dk/bound/9788772003788/margrethe-hjort-hay-2018-margrethes-perler-indbundet-bog) and [price comparison](https://bog.nu/titler/margrethes-perler-margrethe-hay) websites unrelated to the publisher.